shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/viewmodel/handler/feeding/DefaultFeedingHandler.kt

```diff
@@ -82,14 +100,55 @@ class DefaultFeedingHandler(
 
     override fun CoroutineScope.handleFeedingEvent(event: FeedingEvent) {
         when (event) {
-            is Start -> launch { startFeeding(event.id) }
+            is WillFeedClicked -> handleWillFeedClicked(this, event)
```

Comment:
WillFeed is back 👍 

Given `WillFeedDialog` and `WillFeedState` is already added, I would recommend to create its own `WillFeedViewModel`. This way we can isolate WillFeed events and detach them from feeding handler.

Reply:
Done